packages/logger/src/providers/LoggerProvider.ts

```diff
@@ -0,0 +1,126 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/logger
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+declare function require(name:string): any;
+var pino = require('pino');
+var pinoHttp = require('pino-http');
+
+import * as fs from 'fs';
+import * as path from 'path';
+import {Logger} from '../keys';
+import {Provider} from '../../../context';
+import {inject} from '../../../core';
+
+const loggerProviderKey = Logger.System.LOGGER_PROVIDER;
+
+/**
+ * @description setup Pino HTTP logger
+ */
+export class SetupPinoHttpLogger implements Provider<any> {
```

Comment:
Let's leave "pino" from class names please. The fact that we are using pino is an implementation detail IMO. Even the component is called `@loopback/logger`, not `@loopback/pino` or `@loopback/pino-logger`.